<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLZCPropertyDlg</class>
 <widget class="QDialog" name="FLZCPropertyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zone Controler Propertys</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_6">
     <item>
      <property name="text">
       <string>1126</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1225</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1324</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1423</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1522</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1621</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1819</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1918</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2017</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2116</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2215</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2314</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2413</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2611</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Passcode Mid</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="10" column="1" rowspan="2" colspan="3">
    <widget class="QComboBox" name="comboBox_4"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>TIMER_ACFAIL_XMIT_DELAY</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_3"/>
   </item>
   <item row="11" column="0" rowspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>TIMER_SIG_SIL</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Comments</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mac Address</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Passcode Hei</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_5"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>SCU_FEATURE_FLAGS</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_9"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Passcode Low</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="14" column="1">
    <widget class="QPushButton" name="pb_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Pan ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="leMAC">
     <property name="inputMask">
      <string>NN:NN:NN:NN:NN:NN:NN:NN</string>
     </property>
     <property name="text">
      <string>00:00:00:00:00:00:00:00</string>
     </property>
     <property name="maxLength">
      <number>23</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="14" column="3">
    <widget class="QPushButton" name="pb_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="14" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
